Dockerfile

```diff
@@ -1,20 +1,27 @@
-FROM ubuntu:14.04
-MAINTAINER Whit Morriss <whit.morriss@canonical.com>
+FROM ubuntu:16.04
+MAINTAINER Matthew Bruzek <Matthew.Bruzek@canonical.com>
 
-ADD setup.sh /setup.sh
-RUN /setup.sh
+RUN useradd -m ubuntu -s /bin/bash
 
-RUN mkdir /home/ubuntu/.juju
-RUN mkdir /home/ubuntu/trusty
-RUN mkdir /home/ubuntu/precise
+RUN mkdir -p \
+    /home/ubuntu/.local/share/juju \
+    /home/ubuntu/builds \
+    /home/ubuntu/trusty \
+    /home/ubuntu/xenial
 
-VOLUME ["/home/ubuntu/.juju"]
+RUN chown -R ubuntu:ubuntu /home/ubuntu
 
-#ADD patchcontainer.sh /patchcontainer.sh
+VOLUME [ \
```

Comment:
easy kids ;)

and the \ could be avoided by putting the first array element on that top line.

```
VOLUME [ "/foo",
                  "/bar" ]
```

Unless i'm full of lies, then 🌚